docs/api/remote-procedure-calls-blockchain.md

```diff
@@ -1302,6 +1322,10 @@ Name | Type | Presence | Description
 →→→<br>`descendent` | number | Optional<br>(0 or 1) | **Added in Dash Core 0.17.0**<br>Modified fees (see above) of in-mempool descendants (including this one) in DASH
 → →<br>`depends` | array | Required<br>(exactly 1) | An array holding TXIDs of unconfirmed transactions this transaction depends upon (parent transactions).  Those transactions must be part of a block before this transaction can be added to a block, although all transactions may be included in the same block.  The array may be empty
 → → →<br>Depends TXID | string | Optional (0 or more) | The TXIDs of any unconfirmed transactions this transaction depends upon, encoded as hex in RPC byte order
+→ →<br>`spentby` | array | Required<br>(exactly 1) |  **Added in Dash Core 20.0.0**<br>An array of unconfirmed transactions spending outputs from this transaction
+→ → →<br>TXID | string | Optional (0 or more) | The TXIDs of any unconfirmed transactions spending from this transaction
+→ →<br>`unbroadcast` | bool | Required<br>(exactly 1) | **Added in Dash Core 20.0.0**<br>True if this transaction  is currently unbroadcast (initial broadcast not yet acknowledged by any peers)
+→ →<br>`instantlock` | bool | Required<br>(exactly 1) |  **Added in Dash Core 20.0.0**<br><br>Set to `true` if this transaction was locked via InstantSend
```

Comment:
Turns out this has been here for a long time, but it was just missing :see_no_evil: Actually it was mis-named in the help too. So good catch :+1: I opened this PR to fix the naming issue in Core: https://github.com/dashpay/dash/pull/5512. Actually affects `getmempoolentry` and `getmempooldescendants` too.
```suggestion
→ →<br>`instantlock` | bool | Required<br>(exactly 1) | Set to `true` if this transaction was locked via InstantSend
```